src/transformStateWithClones.js

```diff
@@ -7,7 +7,31 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const arrResult = [];
+  const STATE_PREVIOS = { ...state };
```

Comment:
The variable name `STATE_PREVIOS` is not following the naming conventions. It seems to be a typo and should be `statePrevious` to reflect that it holds the previous state. Additionally, using uppercase for non-constant values can be misleading. A better name would be `stateCopy` or `currentState` to convey that it is a mutable copy of the state.